Find a functionally equivalent code:android.widget.CalendarView.getUnfocusedMonthDateColor:COMMENT
Method Modifier: public      
Comment:/**
 * Gets the color for the dates in a not focused month.
 *
 * @return A not focused month date color.
 *
 * @attr ref android.R.styleable#CalendarView_unfocusedMonthDateColor
 */

@@ -1,3 +1,3 @@
 {
-    return mFocusedMonthDateColor;
+    return mDelegate.getUnfocusedMonthDateColor();
 }

